src/redux/categories/categories.reducer.js

```diff
@@ -0,0 +1,31 @@
+import categoriesActionType from './categories.type';
+
+const INITIAL_STATE = {
+  loading: true,
+  categories: [],
+  error: false,
+};
```

Comment:
- [ ] Kindly create and Export **Action Creator**. (`checkStatus` function). Check out this [article](https://medium.com/swlh/few-ways-to-update-a-state-array-in-redux-reducer-f2621ae8061) to learn how to create **Action Creators**.